app/containers/MessageBox/index.js

```diff
@@ -322,6 +323,15 @@ class MessageBox extends Component {
 		if (isTablet) {
 			EventEmiter.removeListener(KEY_COMMAND, this.handleCommands);
 		}
+		if (isAndroid) {
+			BackHandler.removeEventListener('backPress');
```

Comment:
You should pass in the reference to your back function

Reply:
Right, I changed to `this.backHandler = BackHandler.addEventListener('backPress', this.backPress);` and `this.backHandler.remove()`

Reply:
Not sure that you need to add another field `this.backHandler`, `BackHandler.removeEventListener('backPress', this.backPress);` should work.

Reply:
Right work fine, I passed the reference!